joint_trajectory_controller/include/joint_trajectory_controller/trajectory_builder.h

```diff
@@ -93,7 +99,7 @@ class TrajectoryBuilder
    */
   virtual void reset();
 
-public: 
+public:
```

Comment:
Before someone else points this out.  My editor did this as it automatically strips whitespace, I can remove this change if you care.

Reply:
it's a welcome change